assets/src/application/ui/layout/EspressoDropdownMenu/index.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import classnames from 'classnames';
+import { flatMap, isEmpty, isFunction } from 'lodash'; // to be replaced with ramda
+import { DOWN } from '@wordpress/keycodes';
+import { Dropdown, NavigableMenu } from '@wordpress/components';
```

Comment:
This still depends on some other packages.

Reply:
ya that kinda stinks, but whatcha gonna do?
hopefully we can utilize a better solution sometime soon